### ISSUE TITLE
Economy rewards not being applied in v1.3.0/API7

### ISSUE THREAD

**Issue by morrissimo** [pie-flavor/React#7]
When a player wins, the reward commands execute but the configured economy reward isn't actually applied to the player's balance.

Console error when this occurs, React config, and economy (TotalEconomy) config:
https://gist.github.com/anonymous/032fa725de393031a718d1bbd68d2383

Versions:
* forge-1.12.2-14.23.2.2613-universal.jar
* spongeforge-1.12.2-2611-7.1.0-BETA-2931.jar
* React-1.3.0.jar
* TotalEconomy-1.7.1-API_7.jar

**Comment by morrissimo** [comment]
For the time being, I've managed to workaround the issue by disabling the economy reward config altogether and just using a server command to credit the player directly: `*adminpay $winner <amount>`. Figured you'd still want the report in case it's not actually a config pebcak on my end.

**Comment by pie-flavor** [comment]
Can you post the config? Because according to that report, either you haven't specified a currency, or the currency you specified doesn't exist.

**Comment by morrissimo** [comment]
Yep - both the React config and my economy plugin's config are all on that one gist
